media/js/base/send-to-device.js

```diff
@@ -9,29 +9,28 @@ if (typeof window.Mozilla === 'undefined') {
     window.Mozilla = {};
 }
 
-(function($) {
+(function() {
     'use strict';
 
     var SendToDevice = function(id) {
 
-        this.formId = typeof id !== 'undefined' ? '#' + id : '#send-to-device';
+        this.formId = typeof id !== 'undefined' ? id : 'send-to-device';
 
         this.formLoaded = false;
         this.formTimeout = null;
         this.smsEnabled = false;
 
-        this.$widget = $(this.formId);
-        this.$form = this.$widget.find('.send-to-device-form');
-        this.$formFields = this.$form.find('.send-to-device-form-fields');
-        this.$input = this.$formFields.find('.send-to-device-input');
-        this.$thankyou = this.$widget.find('.thank-you');
-        this.$errorList = this.$form.find('.error-list');
-        this.$spinnerTarget = this.$form.find('.loading-spinner');
-        this.$footerLinks = this.$widget.find('footer > ul');
-        this.$sendAnotherLink = this.$form.find('.send-another');
-        this.$formHeading = this.$widget.find('.form-heading');
-        this.spinnerColor = this.$widget.data('spinnerColor') || '#000';
-        this.countries = this.$widget.data('countries');
+        this.$widget = document.getElementById(this.formId);
```

Comment:
The prefix of adding `$` to the start of variable names is from a long gone convention from when we wanted to distinguish between native DOM objects and jQuery objects. Now that we're removing jQuery from this file, I think we're fine to remove `$` from the beggining of the variable names.